src/pages/components/BookmarkPage.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Browser 1.0
+
+Page {
+
+    property string search
+
+    BookmarkModel { id: bookmarkModel }
```

Comment:
Would it make sense to make `BookmarkModel` a singleton? Maybe not, if under normal circumstances no instances of this model gets allocated. But if at least one instance always exists, making it a singleton might be a good idea from the performance prospective.